instr.ml

```diff
@@ -398,12 +404,13 @@ class map = object (m)
       Assert (m#expression e)
     | Stop e ->
       Stop (m#expression e)
-    | Osr {label; cond; target; map} ->
+    | Osr {label; cond; target; var_map; frame_maps} ->
       Osr {
         label;
```

Comment:
fair enough, though I must note that the original #op method did not have access to the arguments and I am really wondering how you would change the operation independently from the args :)

the not having to think and seeing the pieces in isolation was imho the reason for the bug where we converted `x[i]` to `nil[i]`.

I am not arguing against a visitor here. The only thing I am saying is that it is far from clear to me what the granularity of the visits should be and that if I am not sure how to proceed with a feature we don't need currently I leave it to be implemented when a usecase arises.